FileShareClient/FileShareApiClient.cs

```diff
@@ -19,6 +19,7 @@ public interface IFileShareApiClient
         Task<BatchStatusResponse> GetBatchStatusAsync(string batchId);
         Task<BatchSearchResponse> Search(string searchQuery, int? pageSize = null, int? start = null);
         Task<IResult<BatchSearchResponse>> Search(string searchQuery , int? pageSize, int? start , CancellationToken cancellationToken);
+        Task<IResult<BatchAttributesSearchResponse>> BatchAttributeSearchResponse(string searchQuery, CancellationToken cancellationToken);
```

Comment:
Please change the name back to `BatchAttributeSearch`
```suggestion
        Task<IResult<BatchAttributesSearchResponse>> BatchAttributeSearch(string searchQuery, CancellationToken cancellationToken);
```

Reply:
Updated as suggested